paddle/fluid/framework/unused_var_check.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include <string>
+#include <unordered_set>
+#include "paddle/fluid/framework/operator.h"
+
+DECLARE_bool(enable_unused_var_check);
+DECLARE_bool(use_mkldnn);
```

Comment:
Not important: `DECLARE_xxx` here is not necessary in header file. You can move them to `.cc`.

Reply:
I remove `DECLARE_bool(use_mkldnn)` since it is not used, and keep `DECLARE_bool(enable_unused_var_check)` because some `.cc` file will include this header file and use the `FLAGS`.